UIforETW/ChildProcess.cpp

```diff
@@ -214,14 +219,14 @@ void ChildProcess::WaitForCompletion(bool printOutput)
 	if (hChildThread_)
 	{
 		WaitForSingleObject(hChildThread_, INFINITE);
-		CloseHandle(hChildThread_);
+		handle_close::closeHandle(hChildThread_);
 		hChildThread_ = 0;
 	}
 
 	// Clean up.
 	if (hPipe_ != INVALID_HANDLE_VALUE)
 	{
-		CloseHandle(hPipe_);
+		handle_close::closeHandle(hPipe_);

```

Comment:
That sucks that the regular CloseHandle doesn't trigger the same warnings. I'll have to experiment with that.